assets/style.css

```diff
@@ -0,0 +1,603 @@
+}
+
+.heading-img figcaption span {
+  font-size: 13px;
+  line-height: 18px;
+}
+
+.width-class {
+  width: 40%;
+  margin: 0 auto;
+}
+
+.content {
+  font-family: nyt-imperial, georgia, "times new roman", times, serif;
+  font-size: 20px;
+  line-height: 30px;
+  color: #333;
+  font-weight: 400;
+}
+
+.main-content .author-name {
+  margin-top: 50px;
+  margin-bottom: 5px;
+}
+
+.extra-info a {
+  text-decoration: underline;
+}
+
+.space-astronomy a {
+  grid-column: 1/-1;
+  font-size: 16px;
+  line-height: 22px;
+  color: #000;
+  font-weight: 700;
+}
+
+.most-popular a {
+  font-size: 19px;
+  line-height: 23px;
+  font-weight: 500;
+  color: #000;
+  display: inline-block;
+  padding: 10px 0;
+  margin: 10px 0;
+  border-top: 1px solid rgba(211, 204, 204, 0.822);
+  text-decoration: none;
+}
+
+.footer-bottom a {
+  display: inline-block;
+  text-decoration: none;
+  padding: 0 5px;
+  font-size: 11px;
+  color: #666;
+  font-weight: 400;
+}
+
+.footer-heading a {
+  font-size: 13px;
+  text-decoration: none;
+  color: #567b95;
+}
+
+.editor-pick-aside a {
+  color: #333;
+  text-decoration: none;
+  font-size: 14px;
+  line-height: 17px;
+  font-weight: 100;
+}
+
+.content span a {
+  font-weight: 400;
+  color: #326891;
+  text-decoration: underline;
+}
+
+.main-content .author-name a {
+  text-decoration: underline;
+  text-decoration-color: rgba(180, 177, 177, 0.589);
+}
+
+.space-astronomy a:hover {
+  text-decoration: underline;
+  cursor: pointer;
+}
+
+.most-popular a:hover {
+  color: #666;
+}
+
+.footer-bottom a:hover {
+  text-decoration: underline;
+}
+
+.editor-pick-aside a:hover {
+  text-decoration: underline;
+}
+
+.main-content p a:hover {
+  cursor: pointer;
+  text-decoration: none;
+}
+
+.date-share-icons p {
+  font-size: 13px;
+}
+
+.date-share-icons ul {
+  flex: 70%;
+  justify-content: flex-end;
+}
+
+.social-icons {
+  display: inline-block;
+  margin: 0 9px;
+  font-size: 14px;
+  background-color: rgba(209, 208, 208, 0.623);
+  padding: 6px;
+  border-radius: 50%;
+  border: 1px solid rgba(209, 208, 208, 0.623);
+}
+
+.fa-facebook {
+  padding: 6px 9px;
+}
+
+.fa-bookmark,
+.fa-comment-alt {
+  font-size: 25px;
+  padding: 0 7px;
+  color: rgba(0, 0, 0, 0.603);
+  display: inline-block;
+  font-weight: 100;
+}
+
+.book-comment {
+  font-size: 50px;
+  font-weight: 400;
+  line-height: 20px;
+  margin: 0 10px;
+}
+
+.date-share-icons .social-icons:hover {
+  background-color: #fff;
+  cursor: pointer;
+}
+
+.theory-of-inflation {
+  width: 70%;
+  margin: 0 auto;
+}
+
+.theory-of-inflation h4,
+.theory-of-inflation p {
+  font-size: 16px;
+}
+
+.theory-of-inflation h4 {
+  margin-bottom: -8px;
+}
+
+.theory-of-inflation p {
+  font-size: 14px;
+  line-height: 16px;
+  color: rgb(107, 103, 103);
+}
+
+.menu li {
+  font-size: 14px;
+  padding-bottom: 10px;
+}
+
+.subscribe-section li {
+  padding-bottom: 15px;
+  padding-left: -20px;
+}
+
+.first-row ul li,
+.second-row ul li {
+  display: grid;
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

 - [ ] Please add a class to the `<ul>`  element and remove first-row class as parent

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_